src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/EditMask.php

```diff
@@ -713,4 +716,63 @@ class_implements(
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Get the breadcrumb navigation via event.
+     *
+     * @return string
+     *
+     * @SuppressWarnings(PHPMD.Superglobals)
+     * @SuppressWarnings(PHPMD.CamelCaseVariableName)
+     */
+    protected function breadcrumb()
+    {
+        $event = new GetBreadcrumbEvent($this->getEnvironment());
+
+        $this->getEnvironment()->getEventDispatcher()->dispatch(sprintf('%s', $event::NAME), $event);
+
+        $arrReturn = $event->getElements();
+
+        if (!is_array($arrReturn) || count($arrReturn) == 0) {
+            return null;
+        }
+
+        $GLOBALS['TL_CSS'][] = 'system/modules/dc-general/html/css/generalBreadcrumb.css';
+
+        $objTemplate = $this->getTemplate('dcbe_general_breadcrumb');
+        $this->addToTemplate('elements', $arrReturn, $objTemplate);
+
+        return $objTemplate->parse();
+    }
+
+    /**
+     * Create a new instance of ContaoBackendViewTemplate with the template file of the given name.
+     *
+     * @param string $strTemplate Name of the template to create.
+     *
+     * @return ContaoBackendViewTemplate
+     */
+    protected function getTemplate($strTemplate)
```

Comment:
This method is only to be called once and there will not likely be any reusage of it. Please move the call to the code above.